modules/ROOT/pages/policies-policy-level.adoc

```diff
@@ -8,52 +8,50 @@ Policies are by default applied to the entire API, unless configured otherwise.
 
 At the resource level of granularity, policies are applied to only those requests that match the criteria.  All policies, except the Cross-Origin Resource Sharing (CORS) policy, are resource-level.
 
-With Mule 4, resource-level policy support now includes HTTP-based APIs along with RAML APIs, as opposed to support for only RAML APIs in Mule 3. A policy applied at a resource level affects all HTTP methods within the resource: PUT, GET, POST, and selected HTTP methods:
+With Mule 4, resource-level policy support now includes HTTP-based APIs along with RAML APIs, as opposed to support for only RAML APIs in Mule 3. A policy applied at a resource level affects all HTTP methods within the resource: PUT, GET, POST, and selected HTTP methods.
 
-image::methods-resources.png[]
+//image::methods-resources.png[]
 
 You can apply multiple conditions to filter your resources and HTTP methods. The URI template regex can be applied to one, many, or all the methods in your API.
 
-== Usage Scenarios
+//explain here what is URI template regex--fede
 
-The uses for resource level policies are limitless. Listed below are a few examples of how you might implement the resource-level policies:
-The uses for resource-level policies are limitless. A few examples are:
+== Usage Scenarios for Resource-Level Policies
+
+The uses for resource-level policies are limitless. Listed below are a few examples of how you might implement the resource-level policies:
 
 * Applying policies to specific resources
 * Securing a subset of an API
 * Setting different limits on resources
 
-If you have a RAML or OAS spec attached to your API, click Preview Resource Matching to check which resources your filters affect.
-
-//Where does a user click this?
-
-Following is an example API for the usage scenario:
+If you have a RAML or OAS specification attached to your API, you can use the  Preview Resource Matching option to check which resources are affected by your filters:
```

Comment:
GLOBALLY beware of introducing extra spaces:
```suggestion
If you have a RAML or OAS specification attached to your API, you can use the Preview Resource Matching option to determine which resources are affected by your filters:
```

Reply:
Done! Thanks!